eotsmanager/client/rpcclient.go

```diff
@@ -90,6 +90,26 @@ func (c *EOTSManagerGRpcClient) CreateRandomnessPairList(uid, chainID []byte, st
 	return pubRandFieldValList, nil
 }
 
+func (c *EOTSManagerGRpcClient) CreateRandomnessPairListWithInterval(uid, chainID []byte, startHeight uint64, num uint32, interval uint64) ([]*btcec.FieldVal, error) {
+	// For now, implement using existing RPC by calling individual heights
+	// TODO: Later can add dedicated GRPC method for efficiency
+	pubRandList := make([]*btcec.FieldVal, 0, num)
+
+	for i := uint32(0); i < num; i++ {
+		height := startHeight + uint64(i)*interval
+		singleList, err := c.CreateRandomnessPairList(uid, chainID, height, 1)
```

Comment:
Lets leave some comment why we hard code `1` here as it is not immeadiatly obvious to me